### PR TITLE
Fix merchordertest

### PR DESCRIPTION
**Pull request by diegocofre**
mercadopago#62
mercadopago#51 (parcial)

### CODE REVIEW

File: MercadoPagoSDK.Test/Resources/PreferenceTest.cs [hunk 1]
```diff
@@ -17,18 +15,20 @@ public class PreferenceTest
         Preference LastPreference;
 
         [SetUp]
-        public void Init()
+        public void Init()
         {
             // Avoid SSL Cert error
             ServicePointManager.ServerCertificateValidationCallback = delegate { return true; };
+            ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls12;
```
Comment (joelibaceta) [inline MercadoPagoSDK.Test/Resources/PreferenceTest.cs]: No es necesario, ya que la linea superior evita la validación de certificados.

File: px-dotnet/MercadoPagoSDK.csproj [hunk 2]
```diff
@@ -38,21 +41,30 @@
...
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <LangVersion>5</LangVersion>
+    <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
     <DebugType>
     </DebugType>
     <Optimize>true</Optimize>
-    <OutputPath>bin\Release\</OutputPath>
+    <OutputPath>..\..\..\..\binarios\</OutputPath>
```
Comment (joelibaceta) [inline px-dotnet/MercadoPagoSDK.csproj]: ?

File: px-dotnet/packages.config [hunk 3]
```diff
@@ -1,7 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Newtonsoft.Json" version="9.0.1" targetFramework="net35" />
+  <package id="Newtonsoft.Json" version="11.0.2" targetFramework="net35" requireReinstallation="true" />
```
Comment (joelibaceta) [inline px-dotnet/packages.config]: ¿Es necesario este update?

File: MercadoPagoSDK.Test/Resources/PreferenceTest.cs (hunk 1 above)
Comment (diegocofre) [reply MercadoPagoSDK.Test/Resources/PreferenceTest.cs]: Ok. Corregido en el último commit

File: px-dotnet/MercadoPagoSDK.csproj (hunk 2 above)
Comment (diegocofre) [reply px-dotnet/MercadoPagoSDK.csproj]: Error mio. Corregido.

**Comment by MercadoPagoDevelopers** [comment]
Hi @diegocofre ,

First of all, we would like to give our sincere apologies for our belated answer.

Secondly, **we are happy to announce the brand new SDK**, which includes most of the suggestions that the community made in this repository. Therefore, all the existing issues and pull requests will be closed.

We are continuously working to keep our code up to date and communicating to you the latest news.

We hope to receive your comments, issues and contributions to keep moving our product to the next level.

Thanks.
The Mercado Pago Developers team.

**Pull request closed without merging**